components/script/dom/eventsource.rs

```diff
@@ -390,6 +392,14 @@ impl FetchResponseListener for EventSourceContext {
         }
         self.reestablish_the_connection();
     }
+
+    fn process_response_done(&mut self, aborted:bool) {
+        if aborted {
+            // https://html.spec.whatwg.org/multipage/
+            // server-sent-events.html#sse-processing-model:fail-the-connection
```

Comment:
Wrong link? I got https://html.spec.whatwg.org/multipage/server-sent-events.html#sse-processing-model:fail-the-connection-3 which explains what happens when the fetch is aborted.

Reply:
Yep, thanks!